scripts/chk_shellscripts/ignore.txt

```diff
@@ -1,12 +1,5 @@
 ./test/docsCodeStyle.sh
 ./test/cmdlineTests.sh
-./test/externalTests.sh
-./test/externalTests/common.sh
-./test/externalTests/gnosis.sh
-./test/externalTests/zeppelin.sh
-./test/externalTests/colony.sh
-./test/externalTests/solc-js/solc-js.sh
-./scripts/common.sh
```

Comment:
Ah, I wondered why shellcheck ran only for ens.sh and not others. This explains it!